### ISSUE TITLE
Some help with hyperparameters please!

### ISSUE THREAD

**Issue by pablodawson** [hustvl/4DGaussians#96]
Hey, hope you're doing well.
I've been experimenting with your codebase using my own captures, and converting them to the Dynerf structure.
It seems to reconstruct the static scene well (same as standard 3DGS).
However, the dynamic part is often ignored, maybe the movement is too large?
What parameters are the most relevant to mitigate this?
Attached is an example frame: The dynamic part (cyclist in this case) is almost invisible.

I'm guessing _time_smoothness_weight_ should play a part in this, not sure about the rest.
Thanks.

![00042](https://github.com/hustvl/4DGaussians/assets/74978011/3a95b2bc-3039-4c04-ab25-110685d141a4)
![00042](https://github.com/hustvl/4DGaussians/assets/74978011/b7c55f22-fd9f-4c6c-a94f-27784b5b9db5)

**Comment by guanjunwu** [comment]
1. you can enlarge the training iteration such as `iterations = 60000`, `densify_until_iter = 30_000` or more.
2. Actually, 4DGS is hard to learn the large motion such as bicycle, you can check the appendix of my paper.
3. try to use the dense point clouds to initialize the 4D Gaussians.

**Comment by azzarelli** [comment]
I've been stuck on (and still am) avoiding overtraining scenes with sparse views. Through this there's a couple things I think I understand better -> Namely that the regularisers, except from the Plane TV, won't have a significant impact on the final result (unless the weights are really small/large). Essentially, if you are not getting "okay" results without the temporal regularisers, then using them/tuning them wont make a big difference.

As @guanjunwu alludes to, plane based methods are a lot more sensitive to the initialisation. So you may want to consider other methods (such as dust3R) to improve point-cloud initialisation, or perhaps increasing the bounding region and number of initial PCs. Additionally, for cases such as mine, lower learning rate and longer training time can help avoid overfitting (even if it takes very long to train).

**Comment by hsilvaga** [comment]
> I've been stuck on (and still am) avoiding overtraining scenes with sparse views. Through this there's a couple things I think I understand better -> Namely that the regularisers, except from the Plane TV, won't have a significant impact on the final result (unless the weights are really small/large). Essentially, if you are not getting "okay" results without the temporal regularisers, then using them/tuning them wont make a big difference.
> 
> As @guanjunwu alludes to, plane based methods are a lot more sensitive to the initialisation. So you may want to consider other methods (such as dust3R) to improve point-cloud initialisation, or perhaps increasing the bounding region and number of initial PCs. Additionally, for cases such as mine, lower learning rate and longer training time can help avoid overfitting (even if it takes very long to train).

I'm encountering a similar issue with dense pointclouds, namely PCLs coming from Flowmap. Which learning rates did you lower? Was it all of them in general or just the ones related to 4DGS (i.e. deformation_lr / grid_lr)?

**Comment by Harryhaozhu** [comment]
> 嗨，希望你一切都好。 我一直在用自己的视频素材测试你的代码库，并将它们转换成 Dynerf 结构。 静态场景的重建效果似乎不错（和标准的 3DGS 一样）。 但是，动态部分经常被忽略，可能是因为运动幅度太大了？ 哪些参数最能有效解决这个问题？ 附件是一个示例帧：动态部分（本例中是骑自行车的人）几乎看不见。
> 
> 我猜想_time_smoothness_weight_应该与此有关，其他的我就不太确定了。 谢谢。
> 
> ![00042](https://github.com/hustvl/4DGaussians/assets/74978011/3a95b2bc-3039-4c04-ab25-110685d141a4) ![00042](https://github.com/hustvl/4DGaussians/assets/74978011/b7c55f22-fd9f-4c6c-a94f-27784b5b9db5)

请问您解决这个问题了吗，我渲染的结果也几乎看不见
